src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -96,6 +127,7 @@ private StringEscapeCharacterFormatDefinition()
         [Name(ClassificationTypeNames.ClassName)]
         [Order(After = PredefinedClassificationTypeNames.Identifier)]
         [Order(After = PredefinedClassificationTypeNames.Keyword)]
+        [Order(Before = ClassificationTypeNames.StaticSymbol)]
```

Comment:
whats the reason for 'before' here (and on the rest)?

Reply:
The static classification is ordered last so that it trumps any styling given to the specific identifier types. Currently we are limiting the static style to being either bold or not

Reply:
I'll add a comment. =)